ONSdigital/eq-survey-runner
# Eq 320 piping

### Changes

Pipes exercise start and end dates into question text
### How to test

1) Check out the branch
2) Run the unit tests
3) Run the app and check the date period question
4) Create a new token using dev mode with new dates and check that the same questionnaire shows the dates in your token
### Who can test

Anybody except @weapdiv-david
